Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -200,7 +200,7 @@ public void Clear()
             /// </summary>
             public void Set(GameObject hitObject, Vector3 hitPointOnObject, Vector4 hitNormalOnObject, RayStep ray, int rayStepIndex, float rayDistance)
             {
-                raycastHit = default(RaycastHit);
+                raycastHit = default;
```

Comment:
Will this compile with the .NET backend? I recall a while back having trouble using the default value expression because that's only available in [C# 7.1 IIRC](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/default-value-expressions)